### PR TITLE
<DropdownBase/> and <DropdownLayout /> - add "getSelectedOptionId" to testkit

### PR DESCRIPTION
**Pull request by bamai**
### 🔦 Summary
https://jira.wixpress.com/browse/DSM-879
...

### ✅ Checklist

- [ ] 👨‍💻 API change is approved by the UI Infra Developers 
- [ ] 👨‍🎨 UX change is approved by the Design System UX 
- 📚 Change is documented
  - [ ] Story
  - [x] API description
  - [ ] Cheatsheet
  - [ ] Other (explain)
- 🔬 Change is tested
  - [x] Component
  - [ ] Visual test

### CODE REVIEW

**Comment by WixBuildServer** [comment]
### 🤖 Hi, I'm #Janet!

  A deploy preview of your PR was successfully released 🖼.

  You can find it under one of the following URLs:
  * https://preview.wix-style-react.com/pr_6115

File: src/DropdownLayout/DropdownLayout.js [hunk 1]
```diff
@@ -379,11 +380,14 @@ class DropdownLayout extends React.PureComponent {
...
   _getDataAttributes = () => {
     const { visible, dropDirectionUp } = this.props;
+    const { selectedId } = this.state;
 
     return filterObject(
       {
         'data-hook': DATA_HOOKS.CONTENT_CONTAINER,
         [DATA_SHOWN]: visible,
+        [DATA_SELECTED_OPTION_ID]:
+          selectedId === 0 ? `${selectedId}` : selectedId,
```
Comment (TalZa1810) [inline src/DropdownLayout/DropdownLayout.js:390]:
```suggestion
          `${selectedId}`,
```

Add than you can test this for both string and number at the tests file

File: src/DropdownLayout/test/DropdownLayout.spec.js [hunk 2]
```diff
@@ -441,6 +441,18 @@ describe('DropdownLayout', () => {
       ).toBe(true);
     });
 
+    it('should not select an option by default', async () => {
+      const driver = createDriver(<DropdownLayout visible options={options} />);
+      expect(await driver.getSelectedOptionId()).toEqual(null);
+    });
+
+    it('should return the selected option', async () => {
```
Comment (TalZa1810) [inline src/DropdownLayout/test/DropdownLayout.spec.js:449]: add here tests for both id that is a number or a string

**Review by TalZa1810 — CHANGES_REQUESTED** [review]
Hey, 
I've added some few comments. Let me know if something is unclear.

**Review by TalZa1810 — APPROVED** [review]

**Pull request merged**